tgstation.dme

```diff
@@ -2855,7 +2855,6 @@
 #include "code\modules\surgery\plastic_surgery.dm"
 #include "code\modules\surgery\prosthetic_replacement.dm"
 #include "code\modules\surgery\remove_embedded_object.dm"
-#include "code\modules\surgery\stomachpump.dm"

```

Comment:
Why

Reply:
how in the actual fuck did this happen

Reply:
More like how did that happen when I already readded the file 3 TIMES NOW.
I'll fix it when I get back